tests/workflows/test_dependency_binaries_manifest.py

```
@@ -0,0 +1,431 @@
+        missing = tmp_path / "does_not_exist.toml"
+
+        with pytest.raises(FileNotFoundError):
+            _collect_manifest_lines(missing)
+
+    def test_malformed_toml_raises(self, tmp_path: Path) -> None:
+        """Malformed TOML raises an exception."""
+        manifest = _write_manifest(
+            tmp_path / "bad.toml", "not valid [[ toml"
+        )
+
+        with pytest.raises(Exception):
```

Ruff (0.15.9)

[warning] Do not assert blind exception: `Exception`

(B017)